src/6model/reprs/MVMSpeshCandidate.c

```diff
@@ -308,6 +314,7 @@ void MVM_spesh_candidate_add(MVMThreadContext *tc, MVMSpeshPlanned *p) {
             spesh->body.spesh_candidates);
     }
     new_candidate_list[spesh->body.num_spesh_candidates] = candidate;
```

Comment:
```suggestion
```
MVM_ASSIGN_REF does `=`, so this line can be removed.

Reply:
Thanks, removed it.